ui/src/assets/i18n/fr.json

```diff
@@ -427,6 +425,31 @@
+  "requirement_help_model" : "Model type 'model': <ul><li>Si vous sélectionnez un <a href=\"https://ovh.github.io/cds/advanced/advanced.worker.model/\">Worker Model</a>, CDS lancera votre Job dans une instance de celui-ci</li><li><a href=\"https://ovh.github.io/cds/tutorials/tutorials.worker-model-docker-simple/\">Créer un modèle de worker en utilisant une image depuis Docker Hub</a></li><li><a href=\"https://ovh.github.io/cds/tutorials/tutorials.worker-model-docker-customized/\">Créer un modèle de worker avec votre propre image docker</a></li><li><a href=\"https://ovh.github.io/cds/tutorials/tutorials.worker-model-openstack/\">Créer un modèle de worker Openstack</a></li><li><a href=\"https://ovh.github.io/cds/advanced/advanced.worker.model/\">En savoir plus</a></li></ul>",
+  "requirement_help_memory" : "Mode type 'memory': <ul><li>Si vous souhaitez 5Go, entrez la valeur suivante: <b>4096</b></li><li>Le prérequis memory est disponible uniquement avec les <a href=\"https://ovh.github.io/cds/advanced/advanced.worker.model/\">Worker Model</a> de type Docker</li></ul>",
+  "requirement_help_network" : "Mode type 'network': <ul><li>CDS choisira un worker qui pourra atteindre cette IP</li></ul>",
+  "requirement_help_hostname" : "Mode type 'hostname': <ul><li>Ce job sera lancé par un worker possédant ce Hostname</li></ul>",
+  "requirement_help_service" : "Mode type 'service': <ul><li><a href=\"https://ovh.github.io/cds/building-pipelines/building-pipelines.requirements/\">Note sur le Service Requirement</a></li><li><a href=\"https://ovh.github.io/cds/tutorials/tutorials.service-link-requirement-nginx/\">Tutorial - Service Link Requirement Nginx Tutorial</a></li><li><a href=\"https://ovh.github.io/cds/tutorials/tutorials.service-link-requirement-pg/\">Service Link Requirement PostgreSQL</a></li></ul>",
+  "requirement_name_error" : "Requirement name is mandatory and must respect the alphanumeric pattern ([a-zA-Z0-9]*)",
+  "requirement_settings_worker_model": "Voir les paramètres du Worker Model",
+  "requirement_options": "Options :",
+
+  "requirement_placeholder_name_binary" : "",
+  "requirement_placeholder_name_model" : "",
+  "requirement_placeholder_name_memory" : "",
+  "requirement_placeholder_name_network" : "",
+  "requirement_placeholder_name_hostname" : "",
+  "requirement_placeholder_name_service" : "HostnameDuService",
+
+  "requirement_placeholder_value_binary" : "bash",
+  "requirement_placeholder_value_model" : "Choisissez un modèle de worker",
+  "requirement_placeholder_opts_model" : "Si c'est un model de worker Docker, you pouvez ajouter des options. Cf. documentation",
+  "requirement_placeholder_value_memory" : "4096",
+  "requirement_placeholder_value_network" : "127.0.0.1",
+  "requirement_placeholder_value_hostname" : "votreHostname",
+  "requirement_placeholder_value_service" : "postgres:9.5.3",
+  "requirement_placeholder_opts_service" : "POSTGRES_USER=myuser\nPOSTGRES_PASSWORD=mypassword",
 
```

Comment:
en.json finish at line 453. So I guess , 1 translate is missing here

Reply:
fixed